scripts/package.ts

```diff
@@ -14,8 +15,11 @@ const package = async () => {
     await fse.rename('./bin/studio-linux', `./bin/studio-v${version}-linux-x64`)
     await fse.rename('./bin/studio-macos', `./bin/studio-v${version}-darwin-x64`)
   } catch (err) {
-    console.error('Error running: ', err.cmd, '\nMessage: ', err.stderr, err)
+    if (err instanceof Error) {
+      const error = err as ExecException
+      console.error('Error running: ', error.cmd, '\nMessage: ', error['stderr'], err)
+    }
   }
 }
 
-package()
+void packageApp()
```

Comment:
Not sure I understand why you added the void here?

Reply:
Top-level awaits are not permitted, i used void instead of tslint ignore